.github/spelling/known_words_corpus.txt

```diff
@@ -763,3 +763,27 @@ micropayments
 KYC
 Alrighty
 agentverseicon
+24z
```

Comment:
where is spelling finding these?

Reply:
Will edit the script for spell checks possibly to avoid `<ApiEndpointRequestResponse/>` blocks

Reply:
Merged this change, please update this @gautamgambhir97